Engine/CMakeLists.txt

```diff
@@ -64,8 +65,8 @@ elseif ( WIN32 )
 
     else()
         message(FATAL_ERROR "Target platform is not X64 - Compilation denied")
-    endif() # End of Windows
-endif() # End of MAC
```

Comment:
I thought it might be useful, a little bit more information :)

Reply:
Then we need to add this to each if statement eventually that might add clutter

Reply:
Fair enough